<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleDeviceConfiguration</class>
 <widget class="QWidget" name="SingleDeviceConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frmDevice">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lbInDevice_3">
        <property name="text">
         <string>Device:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="cbSoundDevice">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="KPushButton" name="pbAdvancedConfiguration">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Advanced configuration</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="pbTest">
          <property name="text">
           <string comment="Test the current configuration for errors">Test</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="pbRemove">
          <property name="text">
           <string>Remove device</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="wgAdvancedOptions" native="true">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lbChannels_3">
           <property name="text">
            <string>Channels:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KIntNumInput" name="sbChannels">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbSamplerate_3">
           <property name="text">
            <string>Sample-Rate:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QSpinBox" name="sbSampleRate">
             <property name="minimum">
              <number>8000</number>
             </property>
             <property name="maximum">
              <number>128000</number>
             </property>
             <property name="value">
              <number>16000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbHz_3">
             <property name="text">
              <string>Hz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbResample">
           <property name="text">
            <string>Resampling:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="cbResample">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QSpinBox" name="sbResampleSampleRate">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>8000</number>
             </property>
             <property name="maximum">
              <number>128000</number>
             </property>
             <property name="value">
              <number>16000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbHzResample">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Hz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Used for:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="cbTraining">
           <property name="text">
            <string>Training</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QWidget" name="wgRecognition" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="cbRecognition">
              <property name="text">
               <string>Recognition (under the following conditions)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="CompoundConditionSettings" name="wgRecognitionConditions" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lbSampleGroup">
           <property name="text">
            <string>Sample Group:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="KLineEdit" name="leSampleGroup"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>CompoundConditionSettings</class>
   <extends>QWidget</extends>
   <header location="global">simoncontextcoreui/compoundconditionsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbResample</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbResampleSampleRate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbResample</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbHzResample</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAdvancedConfiguration</sender>
   <signal>clicked(bool)</signal>
   <receiver>wgAdvancedOptions</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRecognition</sender>
   <signal>clicked(bool)</signal>
   <receiver>wgRecognitionConditions</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
